source/mir/ndslice/selection.d

```diff
@@ -1713,9 +1713,12 @@ template IotaSlice(size_t N)
 
 // undocumented
 // zero cost variant of `std.range.iota`
+///
 struct IotaMap()
```

Comment:
I don't remember exactly, but you wanted to document it instead of setting it to `private` or `protected`?